scripts/setup.sh

```diff
@@ -100,3 +100,47 @@ else
   # Mac OS.
   export CHROME_BIN="/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
 fi
+
+# Check the version of Python. If its version does not start with 2.7, then
+# try and see whether the 'python2.7' command exists.
+# Borrowed from bhenning's solution for checking python version
+# http://stackoverflow.com/questions/592620
+test_python_version(){
+  EXPECTED_PYTHON_VERSION_PREFIX="2.7"
+  PYTHON_VERSION=$($1 --version 2>&1)
+  if [[ $PYTHON_VERSION =~ Python[[:space:]](.+) ]]; then
+    PYTHON_VERSION=${BASH_REMATCH[1]}
+  else
+    echo "Unrecognizable Python command output: ${PYTHON_VERSION}"
+  fi
+  if [[ "${PYTHON_VERSION}" = "${EXPECTED_PYTHON_VERSION_PREFIX}*" ]]; then
+    #false
+    return 1
+  else 
+    #true
+    return 0
+  fi
+}
+
+#Setting default Python command. Assumes it is already in $PATH
+PYTHON_CMD="python"
+#Check "python" and "python2.7" for version 2.7
```

Comment:
As mentioned earlier, I think this comment could be replaced with the second sentence of the documentation for test_python_version since it describes the intention of the code a little bit better.